style.css

```diff
@@ -307,3 +308,111 @@ body {
   letter-spacing: -0.0525em;
   color: #fff;
 }
+
+#contact_section {
+  background-image: url(img/image_geometry_3.svg);
+  background-repeat: no-repeat;
+  margin-top: 50px;
+  padding-top: 103px;
+}
+
+.contact_title {
+  font-family: "Playfair Display", serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 50px;
+  line-height: 50px;
+  color: #1a2236;
+  margin-left: 18px;
+}
+
+.contact_text {
+  font-family: Poppins, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -1%;
+  font-style: normal;
+  margin-left: 18px;
+  margin-top: 20px;
+  width: 288px;
+}
+
+.form {
+  margin-top: 32px;
+  display: flex;
+  flex-direction: column;
+}
+
+.input1 {
+  width: 288px;
+  height: 52px;
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #dfdfe0;
+  margin-bottom: 8px;
+  margin-left: 18px;
+  padding: 16px;
+}
+
+.input2 {
+  width: 288px;
+  height: 52px;
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #1a2236;
+  margin-bottom: 8px;
+  margin-left: 18px;
+  padding: 16px;
+}
+
+.input3 {
+  width: 288px;
+  height: 112px;
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #dfdfe0;
+  margin-left: 18px;
+  padding: 16px;
+}
+
+.contact_btn {
+  margin-top: 20px;
+}
```

Comment:
- [ ] **Required:** The contact_btn should visually change when the button is hovered over or/and clicked. This is to visually indicate interaction and add some aesthetics to your website. I hope it makes some sense 😏.